systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicScalabilityUtils.java

```diff
@@ -45,8 +45,8 @@ public static void createTopicsViaK8s(String namespaceName, String clusterName,
 
         for (int i = start; i < end; i++) {
             String currentTopicName = topicPrefix + "-" + i;
-            ResourceManager.getInstance().createResourceWithoutWait(KafkaTopicTemplates.topic(
-                clusterName, currentTopicName, numberOfPartitions, numberOfReplicas, minInSyncReplicas, namespaceName).build());
+            ResourceManager.getInstance().createResourceWithoutWait(KafkaTopicTemplates.topic(namespaceName, currentTopicName, clusterName,
+                numberOfPartitions, numberOfReplicas, minInSyncReplicas).build());
```

Comment:
```suggestion
            ResourceManager.getInstance().createResourceWithoutWait(KafkaTopicTemplates.topic(namespaceName, currentTopicName, kafkaClusterName,
                numberOfPartitions, numberOfReplicas, minInSyncReplicas).build());
```

Reply:
I think this can be changed in different PR, I'm just switching the parameters here.